libraries/shared/repository/repository.go

```diff
@@ -76,7 +76,8 @@ func GetCheckedColumnNames(db *postgres.DB) ([]string, error) {
 		`SELECT column_name FROM information_schema.columns
 		WHERE table_schema = 'public'
 			AND table_name = 'checked_headers'
-			AND column_name ~ '_checked';`
```

Comment:
Maybe we should update the expected column names in the test to omit `_checked`?